### PR TITLE
Add dynamic meta description in page Signup and Login

### PR DESCRIPTION
**Pull request by IgnacioRodrigues**
In this commit, we add that a package called "React-Helmet" is a reusable React component that manages changes in the header of the document and with this, the content of the meta description is added in the login and registration page.

### CODE REVIEW

**Comment by andresmoschini** [comment]
Por favor @IgnacioRodrigues tratá de mantener las ramas de features limpias y sin merges internos

image

Si podés acomodá eso rabaseando en lugar de mergear master en tu rama.

**Comment by IgnacioRodrigues** [comment]
> acomodá

**Comment by IgnacioRodrigues** [comment]
Sorry, I did not want to close this pull request.

File: .editorconfig [hunk 1]
```diff
@@ -10,8 +10,4 @@ max_line_length = 100
 indent_size = 2
 
 [*.md]
-trim_trailing_whitespace = false
-
-[*.{js,ts}]
-block_comment_start = /*
-block_comment_end = */
```
Comment (andresmoschini) [inline .editorconfig]: Why did you remove these rules?

**Comment by andresmoschini** [comment]
Oh, there is a lot of unrelated things here.

Please, fix the PR and let us know.

Today I cannot help you on this, but maybe @mvillaseco or @VicThorX could.

File: .editorconfig (hunk 1 above)
Comment (IgnacioRodrigues) [reply .editorconfig]: This change is the commit for revert merge

**Comment by andresmoschini** [comment]
Helmet looks very interesting. 

As part of this task (or another one), we should apply this same to other places where we are updating the title, and we should also apply it to change the language, see: https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L2

Would it be fine?

**Comment by IgnacioRodrigues** [comment]
> Helmet looks very interesting.
> 
> As part of this task (or another one), we should apply this same to other places where we are updating the title, and we should also apply it to change the language, see: https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L2
> 
> Would it be fine?

Okey, added the new change in this pull request.

**Comment by andresmoschini** [comment]
> Okey, added the new change in this pull request.

I understand that you have other responsibilities than this webapp, so, maybe it is a good idea to merge current work and reserve my proposals for another PR.

**Comment by IgnacioRodrigues** [comment]
Ok, I'm going to merge this extraction request and the other changes will create a new pull request. Thanks @andresmoschini

**Comment by andresmoschini** [comment]
:shipit:
